README.md

```diff
@@ -37,7 +37,7 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a player, I need to be able to see how many points I have earned throughout the game.
 - As a player, I need to be able to replay a game with the same initial hand as a new game if I desire.
 - As a player, I need to be informed when I run out of letters in my hand.
-- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is.
+- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is. (*BONUS QUESTION* :trophy: )
```

Comment:
I'm not sure what your intention is here

Reply:
to emphasise if they do spend time on it, I am happy